syncd/syncd_applyview.cpp

```diff
@@ -6348,6 +6722,9 @@ void executeOperationsOnAsic(
 
         SWSS_LOG_TIMER("asic apply");
 
+        if (enableUnittests())
```

Comment:
>enableUnittests [](start = 12, length = 15)

Name it as 'dumpComparision' or 'debugMode'?

Reply:
maybe not, since it maybe used in many other places in the future not only for comparsion